assets/js/base/components/load-more-button/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export const LoadMoreButton = ( { onClick } ) => (
+	<button
+		className="wc-block-load-more"
+		onClick={ onClick }
+	>
+		{ __( 'Load more', 'woo-gutenberg-products-block' ) }
```

Comment:
From an accessibility standpoint, is this descriptive enough?  If there are multiple review blocks in the page (or other usages of this component), will it be clear to the user what the "Load more" button applies to?  Should we allow the option for passing in more descriptive text (i.e. passing in `Load more products` for instance?).  All of these are just questions :).  Doesn't necessarily need to block this.

Reply:
Thank you for pointing out accessibility here. I spent some time doing accessibility testing today and found some areas where we could give a better experience, specially when updating the list of reviews after changing the order or appending new reviews. I will create a PR tomorrow.

I think your proposal regarding the _Load more_ button makes a lot of sense, I would even extend it to the _Order by_ select above. The visible label can still be _Order by_ but the screen reader label could be _Order reviews by_, this way users navigating the page will know what the order by select refers too. How does it sound?

If it's ok, I will handle all this improvements in an independent PR focused in accessibility.

Reply:
> I think your proposal regarding the Load more button makes a lot of sense, I would even extend it to the Order by select above. The visible label can still be Order by but the screen reader label could be Order reviews by, this way users navigating the page will know what the order by select refers too. How does it sound?

Sounds good to me!

> If it's ok, I will handle all this improvements in an independent PR focused on accessibility.

Good plan 👍